support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/subheading.tsx

```diff
@@ -76,28 +73,23 @@ const getSubHeadingCopy = (
 		};
 	};
 
-	const oneOffCopy = shouldShowLargeDonationMessage
-		? 'It’s not every day we receive such a generous amount. We would love to stay in touch. So that we can, please select the add-ons that suit you best. '
```

Comment:
With this change, we never show this “It’s not every day we receive such a generous amount.” bit to single contributors. Is that ok/deliberate?

Reply:
Yes, deliberate change, the new requested copy does not contain this message, no need to keep unused code.